### ISSUE TITLE
Problem exception when I try to use the new version (0.28.6)

### ISSUE THREAD

**Issue by kip-13** [metabase/metabase#7413]
Info
---

**OS**
```CentOS release 6.9 (Final)```
**Metabase**
```Metabase version v0.28.6 (83e3c0d release-0.28.6)```
**Metabase hosting**
```Docker version 1.7.1, build 786b29d/1.7.1```
**DB**
```MySQL Community Server (GPL) 5.7.20```

----

Hi, I try to use the latest version form docker image and when migrations are running one exception stop all the proccess, this is the log:

```bash
04-23 16:41:15 INFO metabase.util :: Loading Metabase...
04-23 16:41:33 INFO util.encryption :: DB details encryption is DISABLED for this Metabase instance. 🔓
See http://www.metabase.com/docs/latest/operations-guide/start.html#encrypting-your-database-connection-details-at-rest for more information.
04-23 16:41:58 INFO metabase.core :: Starting Metabase in STANDALONE mode
04-23 16:41:58 INFO metabase.core :: Launching Embedded Jetty Webserver with config:
 {:port 3000, :host "0.0.0.0"}

04-23 16:41:58 INFO metabase.core :: Starting Metabase version v0.28.6 (83e3c0d release-0.28.6) ...
04-23 16:41:58 INFO metabase.core :: System timezone is 'America/Argentina/Buenos_Aires' ...
04-23 16:42:04 INFO metabase.core :: Setting up and migrating Metabase DB. Please sit tight, this may take a minute...
04-23 16:42:04 INFO metabase.db :: Verifying mysql Database Connection ...
04-23 16:42:06 INFO metabase.db :: Verify Database Connection ...  ✅
04-23 16:42:06 INFO metabase.db :: Running Database Migrations...
Mon Apr 23 16:42:06 ART 2018 WARN: Establishing SSL connection without server's identity verification is not recommended. According to MySQL 5.5.45+, 5.6.26+ and 5.7.6+ requirements SSL connection must be established by default if explicit option isn't set. For compliance with existing applications not using SSL the verifyServerCertificate property is set to 'false'. You need either to explicitly disable SSL by setting useSSL=false, or set useSSL=true and provide truststore for server certificate verification.
04-23 16:42:06 INFO metabase.db :: Setting up Liquibase...
Mon Apr 23 16:42:06 ART 2018 WARN: Establishing SSL connection without server's identity verification is not recommended. According to MySQL 5.5.45+, 5.6.26+ and 5.7.6+ requirements SSL connection must be established by default if explicit option isn't set. For compliance with existing applications not using SSL the verifyServerCertificate property is set to 'false'. You need either to explicitly disable SSL by setting useSSL=false, or set useSSL=true and provide truststore for server certificate verification.
04-23 16:42:06 INFO metabase.db :: Liquibase is ready.
04-23 16:42:06 INFO metabase.db :: Checking if Database has unrun migrations...
04-23 16:42:18 INFO metabase.db :: Database Migrations Current ...  ✅
com.mchange.v2.cfg.DelayedLogItem [ level -> FINE, text -> "The configuration file for resource identifier 'hocon:/reference,/application,/c3p0,/' could not be found. Skipping.", exception -> null]
04-23 16:42:18 INFO db.migrations :: Running all necessary data migrations, this may take a minute.
Mon Apr 23 16:42:18 ART 2018 WARN: Establishing SSL connection without server's identity verification is not recommended. According to MySQL 5.5.45+, 5.6.26+ and 5.7.6+ requirements SSL connection must be established by default if explicit option isn't set. For compliance with existing applications not using SSL the verifyServerCertificate property is set to 'false'. You need either to explicitly disable SSL by setting useSSL=false, or set useSSL=true and provide truststore for server certificate verification.
Mon Apr 23 16:42:18 ART 2018 WARN: Establishing SSL connection without server's identity verification is not recommended. According to MySQL 5.5.45+, 5.6.26+ and 5.7.6+ requirements SSL connection must be established by default if explicit option isn't set. For compliance with existing applications not using SSL the verifyServerCertificate property is set to 'false'. You need either to explicitly disable SSL by setting useSSL=false, or set useSSL=true and provide truststore for server certificate verification.
Mon Apr 23 16:42:18 ART 2018 WARN: Establishing SSL connection without server's identity verification is not recommended. According to MySQL 5.5.45+, 5.6.26+ and 5.7.6+ requirements SSL connection must be established by default if explicit option isn't set. For compliance with existing applications not using SSL the verifyServerCertificate property is set to 'false'. You need either to explicitly disable SSL by setting useSSL=false, or set useSSL=true and provide truststore for server certificate verification.
04-23 16:42:19 INFO db.migrations :: Running data migration 'repopulate-card-read-permissions'...
04-23 16:42:19 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:21 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:21 WARN middleware.expand :: The syntax for datetime-field has changed in MBQL '98. [:datetime-field <field> :as <unit>] is deprecated. Prefer [:datetime-field <field> <unit>] instead.
04-23 16:42:21 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:22 WARN middleware.expand :: The syntax for aggregate fields has changed in MBQL '98. Instead of `[:aggregation 0]`, please use `[:aggregate-field 0]` instead.
04-23 16:42:22 WARN middleware.expand :: The syntax for order-by has changed in MBQL '98. [<field> :ascending/:descending] is deprecated. Prefer [:asc/:desc <field>] instead.
04-23 16:42:24 WARN middleware.expand :: The syntax for aggregate fields has changed in MBQL '98. Instead of `[:aggregation 0]`, please use `[:aggregate-field 0]` instead.
04-23 16:42:24 WARN middleware.expand :: The syntax for order-by has changed in MBQL '98. [<field> :ascending/:descending] is deprecated. Prefer [:asc/:desc <field>] instead.
04-23 16:42:25 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:25 WARN middleware.expand :: The syntax for datetime-field has changed in MBQL '98. [:datetime-field <field> :as <unit>] is deprecated. Prefer [:datetime-field <field> <unit>] instead.
04-23 16:42:25 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:25 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:25 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:26 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:26 WARN middleware.expand :: The syntax for aggregate fields has changed in MBQL '98. Instead of `[:aggregation 0]`, please use `[:aggregate-field 0]` instead.
04-23 16:42:26 WARN middleware.expand :: The syntax for order-by has changed in MBQL '98. [<field> :ascending/:descending] is deprecated. Prefer [:asc/:desc <field>] instead.
04-23 16:42:26 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:26 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:27 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:27 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:27 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:27 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:27 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:28 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:28 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:29 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:29 WARN middleware.expand :: The syntax for datetime-field has changed in MBQL '98. [:datetime-field <field> :as <unit>] is deprecated. Prefer [:datetime-field <field> <unit>] instead.
04-23 16:42:29 WARN middleware.expand :: The syntax for datetime-field has changed in MBQL '98. [:datetime-field <field> :as <unit>] is deprecated. Prefer [:datetime-field <field> <unit>] instead.
04-23 16:42:29 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:29 WARN middleware.expand :: The syntax for aggregate fields has changed in MBQL '98. Instead of `[:aggregation 0]`, please use `[:aggregate-field 0]` instead.
04-23 16:42:29 WARN middleware.expand :: The syntax for order-by has changed in MBQL '98. [<field> :ascending/:descending] is deprecated. Prefer [:asc/:desc <field>] instead.
04-23 16:42:29 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:29 WARN middleware.expand :: The syntax for aggregate fields has changed in MBQL '98. Instead of `[:aggregation 0]`, please use `[:aggregate-field 0]` instead.
04-23 16:42:29 WARN middleware.expand :: The syntax for order-by has changed in MBQL '98. [<field> :ascending/:descending] is deprecated. Prefer [:asc/:desc <field>] instead.
04-23 16:42:30 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:31 WARN middleware.expand :: The syntax for order-by has changed in MBQL '98. [<field> :ascending/:descending] is deprecated. Prefer [:asc/:desc <field>] instead.
04-23 16:42:31 WARN middleware.expand :: The syntax for order-by has changed in MBQL '98. [<field> :ascending/:descending] is deprecated. Prefer [:asc/:desc <field>] instead.
04-23 16:42:31 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:34 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:34 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:35 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:35 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
04-23 16:42:35 WARN middleware.expand :: You shouldn't specify an :and filter with only one subclause.
java.lang.IllegalArgumentException: No implementation of method: :parse-value of protocol: #'metabase.query-processor.middleware.resolve/IParseValueForField found for class: metabase.query_processor.interface.FieldPlaceholder
        at clojure.core$_cache_protocol_fn.invokeStatic(core_deftype.clj:568)
        at clojure.core$_cache_protocol_fn.invoke(core_deftype.clj:560)
        at metabase.query_processor.middleware.resolve$fn__27986$G__27981__27993.invoke(resolve.clj:218)
        at metabase.query_processor.middleware.resolve$value_ph_resolve_field.invokeStatic(resolve.clj:276)
        at metabase.query_processor.middleware.resolve$value_ph_resolve_field.invoke(resolve.clj:270)
        at metabase.query_processor.middleware.resolve$fn__27894$G__27876__27901.invoke(resolve.clj:83)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.core$apply.invokeStatic(core.clj:646)
        at clojure.core$apply.invoke(core.clj:641)
        at metabase.util$rpartial$fn__2828.doInvoke(util.clj:463)
        at clojure.lang.RestFn.invoke(RestFn.java:408)
        at clojure.walk$walk.invokeStatic(walk.clj:48)
        at clojure.walk$postwalk.invokeStatic(walk.clj:52)
        at clojure.walk$postwalk.invoke(walk.clj:52)
        at clojure.core$partial$fn__4759.invoke(core.clj:2515)
        at clojure.core$map$fn__4785.invoke(core.clj:2644)
        at clojure.lang.LazySeq.sval(LazySeq.java:40)
        at clojure.lang.LazySeq.seq(LazySeq.java:49)
        at clojure.lang.RT.seq(RT.java:521)
        at clojure.lang.LazilyPersistentVector.create(LazilyPersistentVector.java:44)
        at clojure.core$vec.invokeStatic(core.clj:377)
        at clojure.walk$walk.invokeStatic(walk.clj:45)
        at clojure.walk$postwalk.invokeStatic(walk.clj:52)
        at clojure.walk$postwalk.invoke(walk.clj:52)
        at clojure.core$partial$fn__4759.invoke(core.clj:2515)
        at clojure.walk$walk$fn__7806.invoke(walk.clj:48)
        at clojure.core.protocols$iter_reduce.invokeStatic(protocols.clj:49)
        at clojure.core.protocols$fn__6742.invokeStatic(protocols.clj:75)
        at clojure.core.protocols$fn__6742.invoke(protocols.clj:75)
        at clojure.core.protocols$fn__6684$G__6679__6697.invoke(protocols.clj:13)
        at clojure.core$reduce.invokeStatic(core.clj:6545)
        at clojure.walk$walk.invokeStatic(walk.clj:48)
        at clojure.walk$postwalk.invokeStatic(walk.clj:52)
        at clojure.walk$postwalk.invoke(walk.clj:52)
        at clojure.core$partial$fn__4759.invoke(core.clj:2515)
        at clojure.core$map$fn__4785.invoke(core.clj:2644)
        at clojure.lang.LazySeq.sval(LazySeq.java:40)
        at clojure.lang.LazySeq.seq(LazySeq.java:49)
        at clojure.lang.RT.seq(RT.java:521)
        at clojure.core$seq__4357.invokeStatic(core.clj:137)
        at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:24)
        at clojure.core.protocols$fn__6738.invokeStatic(protocols.clj:75)
        at clojure.core.protocols$fn__6738.invoke(protocols.clj:75)
        at clojure.core.protocols$fn__6684$G__6679__6697.invoke(protocols.clj:13)
        at clojure.core$reduce.invokeStatic(core.clj:6545)
        at clojure.core$into.invokeStatic(core.clj:6610)
        at clojure.walk$walk.invokeStatic(walk.clj:49)
        at clojure.walk$postwalk.invokeStatic(walk.clj:52)
        at clojure.walk$postwalk.invoke(walk.clj:52)
        at clojure.core$partial$fn__4759.invoke(core.clj:2515)
        at clojure.core$map$fn__4785.invoke(core.clj:2644)
        at clojure.lang.LazySeq.sval(LazySeq.java:40)
        at clojure.lang.LazySeq.seq(LazySeq.java:49)
        at clojure.lang.RT.seq(RT.java:521)
        at clojure.lang.LazilyPersistentVector.create(LazilyPersistentVector.java:44)
        at clojure.core$vec.invokeStatic(core.clj:377)
        at clojure.walk$walk.invokeStatic(walk.clj:45)
        at clojure.walk$postwalk.invokeStatic(walk.clj:52)
        at clojure.walk$postwalk.invoke(walk.clj:52)
        at clojure.core$partial$fn__4759.invoke(core.clj:2515)
        at clojure.walk$walk$fn__7806.invoke(walk.clj:48)
        at clojure.core.protocols$iter_reduce.invokeStatic(protocols.clj:49)
        at clojure.core.protocols$fn__6742.invokeStatic(protocols.clj:75)
        at clojure.core.protocols$fn__6742.invoke(protocols.clj:75)
        at clojure.core.protocols$fn__6684$G__6679__6697.invoke(protocols.clj:13)
        at clojure.core$reduce.invokeStatic(core.clj:6545)
        at clojure.walk$walk.invokeStatic(walk.clj:48)
        at clojure.walk$postwalk.invokeStatic(walk.clj:52)
        at clojure.walk$postwalk.invoke(walk.clj:52)
        at clojure.core$partial$fn__4759.invoke(core.clj:2515)
        at clojure.core$map$fn__4785.invoke(core.clj:2644)
        at clojure.lang.LazySeq.sval(LazySeq.java:40)
        at clojure.lang.LazySeq.seq(LazySeq.java:49)
        at clojure.lang.RT.seq(RT.java:521)
        at clojure.lang.LazilyPersistentVector.create(LazilyPersistentVector.java:44)
        at clojure.core$vec.invokeStatic(core.clj:377)
        at clojure.walk$walk.invokeStatic(walk.clj:45)
        at clojure.walk$postwalk.invokeStatic(walk.clj:52)
        at clojure.walk$postwalk.invoke(walk.clj:52)
        at clojure.core$partial$fn__4759.invoke(core.clj:2515)
        at clojure.core$map$fn__4785.invoke(core.clj:2646)
        at clojure.lang.LazySeq.sval(LazySeq.java:40)
        at clojure.lang.LazySeq.seq(LazySeq.java:49)
        at clojure.lang.Cons.next(Cons.java:39)
        at clojure.lang.RT.next(RT.java:688)
        at clojure.core$next__4341.invokeStatic(core.clj:64)
        at clojure.core.protocols$fn__6755.invokeStatic(protocols.clj:168)
        at clojure.core.protocols$fn__6755.invoke(protocols.clj:124)
        at clojure.core.protocols$fn__6710$G__6705__6719.invoke(protocols.clj:19)
        at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
        at clojure.core.protocols$fn__6738.invokeStatic(protocols.clj:75)
        at clojure.core.protocols$fn__6738.invoke(protocols.clj:75)
        at clojure.core.protocols$fn__6684$G__6679__6697.invoke(protocols.clj:13)
        at clojure.core$reduce.invokeStatic(core.clj:6545)
        at clojure.core$into.invokeStatic(core.clj:6610)
        at clojure.walk$walk.invokeStatic(walk.clj:49)
        at clojure.walk$postwalk.invokeStatic(walk.clj:52)
        at clojure.walk$postwalk.invoke(walk.clj:52)
        at clojure.core$partial$fn__4759.invoke(core.clj:2515)
        at clojure.core$map$fn__4785.invoke(core.clj:2644)
        at clojure.lang.LazySeq.sval(LazySeq.java:40)
        at clojure.lang.LazySeq.seq(LazySeq.java:49)
        at clojure.lang.RT.seq(RT.java:521)
        at clojure.lang.LazilyPersistentVector.create(LazilyPersistentVector.java:44)
        at clojure.core$vec.invokeStatic(core.clj:377)
        at clojure.walk$walk.invokeStatic(walk.clj:45)
        at clojure.walk$postwalk.invokeStatic(walk.clj:52)
        at clojure.walk$postwalk.invoke(walk.clj:52)
        at clojure.core$partial$fn__4759.invoke(core.clj:2515)
        at clojure.core$map$fn__4785.invoke(core.clj:2646)
        at clojure.lang.LazySeq.sval(LazySeq.java:40)
        at clojure.lang.LazySeq.seq(LazySeq.java:49)
        at clojure.lang.Cons.next(Cons.java:39)
        at clojure.lang.RT.next(RT.java:688)
        at clojure.core$next__4341.invokeStatic(core.clj:64)
        at clojure.core.protocols$fn__6755.invokeStatic(protocols.clj:168)
        at clojure.core.protocols$fn__6755.invoke(protocols.clj:124)
        at clojure.core.protocols$fn__6710$G__6705__6719.invoke(protocols.clj:19)
        at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
        at clojure.core.protocols$fn__6738.invokeStatic(protocols.clj:75)
        at clojure.core.protocols$fn__6738.invoke(protocols.clj:75)
        at clojure.core.protocols$fn__6684$G__6679__6697.invoke(protocols.clj:13)
        at clojure.core$reduce.invokeStatic(core.clj:6545)
        at clojure.core$into.invokeStatic(core.clj:6610)
        at clojure.walk$walk.invokeStatic(walk.clj:49)
        at clojure.walk$postwalk.invokeStatic(walk.clj:52)
        at clojure.walk$postwalk.invoke(walk.clj:52)
        at metabase.query_processor.middleware.resolve$resolve_fields.invokeStatic(resolve.clj:342)
        at metabase.query_processor.middleware.resolve$resolve_fields.invoke(resolve.clj:325)
        at metabase.query_processor.middleware.resolve$resolve.invokeStatic(resolve.clj:411)
        at metabase.query_processor.middleware.resolve$resolve.invoke(resolve.clj:408)
        at metabase.query_processor.middleware.resolve$resolve_middleware$fn__28069.invoke(resolve.clj:422)
        at clojure.core$comp$fn__4727.invoke(core.clj:2460)
        at metabase.query_processor.middleware.expand$expand_middleware$fn__29769.invoke(expand.clj:601)
        at clojure.core$comp$fn__4727.invoke(core.clj:2460)
        at clojure.core$comp$fn__4727.invoke(core.clj:2460)
        at clojure.core$comp$fn__4727.invoke(core.clj:2460)
        at metabase.models.card$mbql_permissions_path_set.invokeStatic(card.clj:174)
        at metabase.models.card$mbql_permissions_path_set.doInvoke(card.clj:137)
        at clojure.lang.RestFn.invoke(RestFn.java:464)
        at metabase.models.card$query_perms_set.invokeStatic(card.clj:200)
        at metabase.models.card$query_perms_set.doInvoke(card.clj:191)
        at clojure.lang.RestFn.invoke(RestFn.java:464)
        at metabase.models.card$maybe_update_read_permissions.invokeStatic(card.clj:315)
        at metabase.models.card$maybe_update_read_permissions.invoke(card.clj:302)
        at metabase.models.card$pre_update.invokeStatic(card.clj:351)
        at metabase.models.card$pre_update.invoke(card.clj:349)
        at clojure.core$comp$fn__4727.invoke(core.clj:2460)
        at toucan.models$fn__3985$G__3904__3990.invoke(models.clj:178)
        at toucan.models$do_pre_update.invokeStatic(models.clj:330)
        at toucan.models$do_pre_update.invoke(models.clj:325)
        at toucan.db$update_BANG_.invokeStatic(db.clj:479)
        at toucan.db$update_BANG_.invoke(db.clj:462)
        at metabase.db.migrations$repopulate_card_read_permissions$fn__32984.invoke(migrations.clj:396)
        at clojure.core$run_BANG_$fn__7276.invoke(core.clj:7393)
        at clojure.core$completing$fn__6948.invoke(core.clj:6585)
        at clojure.core$map$fn__4781$fn__4782.invoke(core.clj:2633)
        at clojure.java.jdbc$init_reduce_rs.invokeStatic(jdbc.clj:1120)
        at clojure.java.jdbc$init_reduce_rs.invoke(jdbc.clj:1112)
        at clojure.java.jdbc$reducible_result_set_STAR_$reify__1874.reduce(jdbc.clj:1142)
        at clojure.core$reduce.invokeStatic(core.clj:6544)
        at clojure.core$reduce.invoke(core.clj:6527)
        at clojure.java.jdbc$query_reducer$fn__1880$fn__1883.invoke(jdbc.clj:1177)
        at clojure.java.jdbc$execute_query_with_params.invokeStatic(jdbc.clj:994)
        at clojure.java.jdbc$execute_query_with_params.invoke(jdbc.clj:987)
        at clojure.java.jdbc$db_query_with_resultset_STAR_.invokeStatic(jdbc.clj:1016)
        at clojure.java.jdbc$db_query_with_resultset_STAR_.invoke(jdbc.clj:996)
        at clojure.java.jdbc$reducible_query$reify__1895.reduce(jdbc.clj:1272)
        at clojure.core$transduce.invokeStatic(core.clj:6600)
        at clojure.core.Eduction.reduce(core.clj:7360)
        at clojure.core$reduce.invokeStatic(core.clj:6544)
        at clojure.core$run_BANG_.invokeStatic(core.clj:7388)
        at clojure.core$run_BANG_.invoke(core.clj:7388)
        at metabase.db.migrations$repopulate_card_read_permissions.invokeStatic(migrations.clj:394)
        at metabase.db.migrations$repopulate_card_read_permissions.invoke(migrations.clj:393)
        at metabase.db.migrations$run_migration_if_needed_BANG_.invokeStatic(migrations.clj:50)
        at metabase.db.migrations$run_migration_if_needed_BANG_.invoke(migrations.clj:41)
        at metabase.db.migrations$run_all_BANG_.invokeStatic(migrations.clj:70)
        at metabase.db.migrations$run_all_BANG_.invoke(migrations.clj:64)
        at clojure.lang.Var.invoke(Var.java:375)
        at metabase.db$run_data_migrations_BANG_.invokeStatic(db.clj:390)
        at metabase.db$run_data_migrations_BANG_.invoke(db.clj:385)
        at metabase.db$setup_db_BANG_.invokeStatic(db.clj:401)
        at metabase.db$setup_db_BANG_.doInvoke(db.clj:392)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at metabase.core$init_BANG_.invokeStatic(core.clj:154)
        at metabase.core$init_BANG_.invoke(core.clj:133)
        at metabase.core$start_normally.invokeStatic(core.clj:258)
        at metabase.core$start_normally.invoke(core.clj:251)
        at metabase.core$_main.invokeStatic(core.clj:279)
        at metabase.core$_main.doInvoke(core.clj:274)
        at clojure.lang.RestFn.invoke(RestFn.java:397)
        at clojure.lang.AFn.applyToHelper(AFn.java:152)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at metabase.core.main(Unknown Source)
04-23 16:42:36 ERROR metabase.core :: Metabase Initialization FAILED:  No implementation of method: :parse-value of protocol: #'metabase.query-processor.middleware.resolve/IParseValueForField found for class: metabase.query_processor.interface.FieldPlaceholder
04-23 16:42:36 INFO metabase.core :: Metabase Shutting Down ...
04-23 16:42:36 INFO metabase.core :: Metabase Shutdown COMPLETE
```

Can someone help me with this ? Thanks!

BTW: Great performance, UI/UX and really good features, thanks for this amazing tool !

**Comment by camsaul** [comment]
Hi @kip-13, sorry you're running into this error. This was fixed by #7313 which is shipping as part of our upcoming 0.29 release, which we're planning on shipping within the next week or so. In the meantime you'll probably want to continue running the prior version of Metabase you were running

**Comment by camsaul** [comment]
Fixed by #7313

**Comment by kip-13** [comment]
Thanks for quickly response, I dont have problem with use a prior version, I've waiting the next release and the good features of this.
